doc/lua_api.txt

```diff
@@ -4790,6 +4790,22 @@ Environment access
     * `pos`: The position where to measure the light.
     * `timeofday`: `nil` for current time, `0` for night, `0.5` for day
     * Returns a number between `0` and `15` or `nil`
+    * `nil` is returned e.g. when the map isn't loaded at `pos`
+* `minetest.get_natural_light(pos[, timeofday])`
+    * Figures out the sunlight (or moonlight) value at pos at the given time of
+      day.
+    * `pos`: The position of the node
+    * `timeofday`: `nil` for current time, `0` for night, `0.5` for day
+    * Returns a number between `0` and `15` or `nil`
+    * This function tests 203 nodes in the worst case, which happens very
+      unlikely
+* `minetest.get_artificial_light(param1)`
```

Comment:
I first thought `minetest.get_artificial_light` worked like `minetest.get_natural_light` (same args). I think it's a bit misleading but if other dev thinks it's ok, I'm ok too.

Reply:
The purpose of this function is to hide the implementation of lighting from mods so that there are no compatibility problems if the lighting system in Minetest changes (`math.floor(param1 / 16)` is very low-level). It is likely that mods use this function together with minetest.get_node; for example, they could test if there is air and no light from a lamp, so for performance reasons it has param1 instead of pos as parameter. 
I also think it's a bit misleading; I could rename the function to `minetest.get_artificial_light_for_param1`, but this would be a very long name for a simple function.

Reply:
Anyway I do not have any alternative name to propose.